src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -1513,6 +1513,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return False
         End Function
 
+        Public Function IsNamespaceDeclaration(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsNamespaceDeclaration
```

Comment:
hrmm... not certain how much this models what we normally do.  shouldnt' this be checking if this is SyntaxKind.NamespaceBloc?

Reply:
Probably :)